control-plane/rest/tests/v0_test.rs

```diff
@@ -335,6 +363,32 @@ async fn client_test(mayastor: &NodeId, test: &ComposeTest, auth: &bool) {
             .unwrap()
     );
 
+    let volume = client
+        .volumes_api()
+        .put_volume_target(&volume.uuid.to_string(), None, None)
```

Comment:
Should we also test the case where the node is specified i.e. assert that it is published on the specified node?

Reply:
well we didn't specify a node (None) - one is auto-selected

Reply:
But should we also check the behaviour when one is explicitly specified?